.github/workflows/conda_build.yml

```
@@ -194,7 +204,12 @@ jobs:
           conda install \
             --channel bioconda \
             --channel conda-forge \
-            --channel $BUILDDIR \
+            --channel nlesc \
+            matchms
+          conda install \
```

Add reminder why this is done
```suggestion
          # Install spec2vec without nlesc channel to prevent package to be installed from nlesc channel instead of wanted $BUILDDIR channel 
          conda install \
```

---

Thanks, that will indeed help to understand this in the future :)